packages/store/tests/action.spec.ts

```diff
@@ -827,17 +827,17 @@ describe('Action', () => {
           'cancellableAction(1) - start',
           'action1 obs - subscribe',
           'Action 2 - dispatching',
-          'Action 1 - dispatch complete',
           'action1 obs - unsubscribe',
+          'Action 1 - dispatch complete',
```

Comment:
Is this change because we unsubscribe before subscribing the real subscriber.
Would it go back to what it was if we unsubscribed after adding the real subscriber (in the fallback operator) ?

Reply:
This change is related to RxJS upgrade, seems like the sequence of some stuff has changed internally